redisinsight/ui/src/pages/rdi/statistics/header/RdiStatisticsHeader.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+
+import InsightsTrigger from 'uiSrc/components/insights-trigger'
+import Header from 'uiSrc/pages/rdi/components/header'
+
+const RdiStatisticsHeader = () => <Header actions={<InsightsTrigger source="statistics page" />} />
```

Comment:
Maybe better to use ExplorePanelTemplate here? anyway we need to think about templates for rdi pages. But need final mockup for it

Reply:
going to remove `ExplorePanelTemplate` for now since it will be implemented in a later card